src/scss/lib/_logger.scss

```diff
@@ -0,0 +1,22 @@
+/**
+ * Throw Error if $keys is not in map
+ *
+ * @param {Map} $map - A map
+ * @param {Args} $keys - Keys in map
+ * @return {Sass::Script::Value::Base}
+ * @throws {Error}
+ *
+ * @access public
+ */
+@function mapGetOrDie($map, $keys...) {
```

Comment:
Essendo questa una `map-get` non la metterei più nel `logger`, ma in un modulo `lib/map`